src/customloaderconfig.hpp

```diff
@@ -103,7 +108,27 @@ class CustomLoaderConfig {
          * @param libraryPath
          */
     void setLibraryPath(const std::string& libraryPath) {
-        this->libraryPath = libraryPath;
+        if (!FileSystem::isLocalFilesystem(libraryPath)) {
+            // Cloud filesystem
+            this->libraryPath = libraryPath;
+        } else if (libraryPath.at(0) == '/') {
+            // Full path case
+            this->libraryPath = libraryPath;
+        } else {
+            // Relative path case
+            if (this->rootDirectoryPath.empty())
+                SPDLOG_ERROR("Using library relative path without setting configuration directory path.");

```

Comment:
Shouldn't we throw exception here or return error status?

Reply:
Ok